modules/common/numbers.ts

```diff
@@ -0,0 +1,25 @@
+import { formatUnits } from 'viem';
+
+/**
+ * Converts wei to float
+ *
+ * @param value
+ * @param decimals
+ * @returns
+ */
+export const fn = (value: string | bigint, decimals: number): number => {
```

Comment:
why call it `fn`? Name it properly :D

Reply:
anything else was too long.. only one letter functions from now on :)